### PR TITLE
docs: Add wiki.

### PR DESCRIPTION
**Pull request by JoeGruffins**
This adds the wiki as is to the docs folder. Suggested by @xaur in #1641

Created with the command:
```
git subtree add --prefix docs/wiki/ https://github.com/JoeGruffins/dcrdex.wiki.git master --squash
```
Using my repo, but should be fine at the same comment 3f0a68ea2897c54c64c9d859b7b7c5575fd4387e

After these changes, collaborators can update the wiki here with pr's, as apposed to just bugging chapp. chapp/buck can update the wiki at any time with commands similar to:
```
git remote add wiki https://github.com/decred/dcrdex.wiki.git
git subtree push --prefix docs/wiki wiki master
``` 
I think.

### CODE REVIEW

**Comment by JoeGruffins** [comment]
Was the wiki originally in the repo's root? Why was it removed? I can't remember now.

**Comment by JoeGruffins** [comment]
Whoops, pushed some irrelevant commits. Fixed now.

File: docs/wiki/Home.md
Comment (chappjc) [inline docs/wiki/Home.md:17]: I guess most of the Home page describing how to contribute to the wiki is incorrect with this change.

Now it's just something like: follow the instructions in docs/wiki/Contribution-Guide.md to clone the dcrdex repository and start a branch for your edits, edit the documents in `docs/wiki`, create a PR with those changes, and once it is merged and administrator will push the changes to the live GitHub Wiki.

Comment (JoeGruffins) [reply docs/wiki/Home.md:17]: I've updated the instructions in #1715 what do you think about those?

**Comment by JoeGruffins** [comment]
just rebased

File: docs/wiki/Home.md
Comment (chappjc) [reply docs/wiki/Home.md:17]: Oh ok so images/wiki-creation.png would still be used.  That makes sense for previewing in your own fork's Wiki.  I have a feeling I'd just do my best to use local previews on my computer's editor for markdown, just not mediawiki.  But maybe I'd get comfy with force pushing the docs/wiki subtree to my wiki remote all the time.

Comment (chappjc) [reply docs/wiki/Home.md:17]: I suppose I'd have a remote called `thewiki` for `https://github.com/decred/dcrdex.wiki.git`, and another called `mywiki` for `https://github.com/chappjc/dcrdex.wiki.git` to make sure I don't get them mixed up if I'm making a wiki PR for instance.

**Review by chappjc — APPROVED** [review]

File: docs/wiki/Home.md
Comment (JoeGruffins) [reply docs/wiki/Home.md:17]: For someone with write access I guess so... I haven't actually tried yet with `https://github.com/decred/dcrdex.wiki.git` I wonder if I can write there because I'm a contributor? I'm not asking for write permissions, just wondering.

> I have a feeling I'd just do my best to use local previews on my computer's editor for markdown

Your IDE?

Comment (JoeGruffins) [reply docs/wiki/Home.md:17]: Anyway, do you want me to change the instructions in this pr? I was just planning on doing the move here but can move the instructions comment here.

**Comment by JoeGruffins** [comment]
@buck54321 any opinion on this pr?

**Comment by chappjc** [comment]
It's a pretty much direct import of the wiki.  We can `git mv` things around if we don't like where things are located.

File: docs/wiki/Home.md
Comment (chappjc) [reply docs/wiki/Home.md:17]: Yeah, IDE or other markdown enabled editor like ReText.  There's a million of them.

Misses your last question, but I think this PR establishes the baseline docs/wiki folder that exactly matches the wiki at dcrdex.wiki.git, so it's perfect as-is.

**Pull request merged**
